config/images.yml

```diff
@@ -1,153 +1,158 @@
 ---
```

Comment:
@jenhadfield-dfe @mariosh-e Let's have a separate conversation about whether we want to bring alt text back in - our last audit suggested marking hero images as decorative but other guidance that we've had suggests the opposite so we might want to revisit